state/state.go

```diff
@@ -131,6 +132,7 @@ func (s *State) getAccount(addr thor.Address) (*Account, error) {
 	if err != nil {
 		return nil, err
 	}
+	metricAccountCounter().AddWithLabel(1, map[string]string{"type": "read", "target": "stackedmap"})
```

Comment:
The `stackedmap` is an in-memory cache stuff with stack functionality, I don't see a necessity of adding metrics for it.

Reply:
Ok, will get rid of the `stackedmap` metrics then, thanks.